subsys/compress/Kconfig

```
@@ -0,0 +1,100 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NRF_COMPRESS
+	bool "nRF compression/decompression library"
+
+if NRF_COMPRESS
+
+config NRF_COMPRESS_COMPRESSION
+	bool
```

Should this have a name?
```suggestion
	bool "Compression support"
```

---

No, placeholder for potential future use